### PR TITLE
New population point dataset

### LINKED ISSUE #64: Create a nicer-looking grid of origins/destinations for basic-usage.ipynb
Currently, the test destinations and origins used in basic-usage.ipynb are created on the fly in a ’regular’ grid of 0.005 degrees. Because Helsinki is so far North, these distances are distorted in the Web Mercator projection used by gdf.explore(). 

Not really sure what to do about it, since a more complex way of generating the sample data set would distract from the actual task. Maybe we could simply supply a ready-made point data set as sample data?

### PR DESCRIPTION
**Pull request by HTenkanen**
Includes:
- New population point dataset for documentation purposes bundled in `docs/data/population_points_2020.gpkg`
- Script file to fetch, manipulate and export the given data from HSY WFS service to `docs/data` directory
- Readme file with descriptions to all datasets bundled with the package at `docs/data/README.md`
- Updated basic_usage.ipynb file with the new data.

Could you @christophfink check the Readme descriptions whether they are correct (I made some guesses there about the origins of the datasets + what you might have done with them) and comment if I guessed wrongly somewhere?  

Resolves #64

### CODE REVIEW

File: docs/basic-usage.ipynb [hunk 1]
```diff
@@ -101,29 +101,14 @@
...
    "outputs": [],
    "source": [
-    "import itertools\n",
     "import geopandas\n",
-    "import numpy\n",
     "\n",
-    "south_west = (24.91, 60.19)\n",
-    "north_east = (24.97, 60.24)\n",
+    "# Filepath\n",
+    "fp = \"data/population_points_2020.gpkg\"\n",
```
Comment (christophfink) [inline docs/basic-usage.ipynb:107]: I’m not sure we need to define the file path in a separate line. Would it be enough to straight call `geopandas.read_file("data/population_points_2020.gpkg")`

File: docs/basic-usage.ipynb [hunk 2]
```diff
@@ -91,7 +91,7 @@
     "\n",
     "A travel time matrix is a data set detailing the travel costs (e.g., time) from each point to each point in a set of origins and destinations in a study area. \n",
     "\n",
-    "For the sake of this exercise, we will create a regular grid of points covering parts of Helsinki."
+    "For the sake of this exercise, we have prepared a grid of points covering parts of Helsinki. The point dataset has been obtained from [Helsinki Region Environmental Services](https://www.hsy.fi/en/environmental-information/open-data/avoin-data---sivut/population-grid-of-helsinki-metropolitan-area/) (HSY), and it also contains information about the number of citizens living within a 250 meter grid cells. "
```
Comment (christophfink) [inline docs/basic-usage.ipynb:94]: Could we add ’under a Creative Commons By Attribution 4.0 license’, and make it ’residents of each cell’ (they (we!) are not necessarily citizens ;) )

**Comment by christophfink** [comment]
I still also pushed some commits with bigger changes to the README and the script (I made it also download a grid cell data set, I guess we can live with 100KiB more in the package?)

**Review by christophfink — CHANGES_REQUESTED** [review]
Some minor things in the basic-usage.ipynb. Note that I made bigger changes to the README and the script, please check those for me :)

File: docs/basic-usage.ipynb (hunk 1 above)
Comment (HTenkanen) [reply docs/basic-usage.ipynb:107]: Sure, we can do this

File: docs/basic-usage.ipynb (hunk 2 above)
Comment (HTenkanen) [reply docs/basic-usage.ipynb:94]: Sure, will modify accordingly.

**Review by HTenkanen — COMMENTED** [review]
Checked your changes, and they look good to me. Will still change that filepath thingy.

**Comment by HTenkanen** [comment]
@christophfink Modified accordingly, please check.

**Review by christophfink — APPROVED** [review]
Looks good, to me. 

There are still a lot of things to improve about this, but the new data set integrates very well now, and visualising the population adds a very nice touch.

**Pull request merged**